data/human/jobs.txt

```diff
@@ -3042,3 +3042,129 @@ mission "Bounty Hunting (Marauder X)"
 	on complete
 		payment 750000
 		dialog phrase "generic fleet bounty hunting payment dialog"
+
+
+
+mission "Hunted Warning"
+	invisible
+	landing
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		attributes "rim" "south" "north" "dirt belt" "core" "frontier"
+	to offer
+		"combat rating" > 8103
+		or
+			not "chosen sides"
+			has "main plot completed"
+	on offer
+		log `Got noticed at a spaceport bar as a "famous warrior." Was warned to watch for pirates looking to make a name for themselves or bounty hunters looking to claim a bounty.`
+		conversation
+			`You walk into a bar in the spaceport looking for something to eat when a man sitting at a table in the corner looks up and takes notice of you. "It's Captain <last>, the famous warrior! Barkeep, give <first> a drink on my tab."`
+			choice
+				`	"Glad to get some recognition. I work hard out here."`
+					goto recognition
+				`	"Famous warrior?"`
+			`	"Of course! Word travels fast in the galaxy, and word on the hyperspace lanes is that you're quite the fighter."`
+			
+			branch recognition
+				"reputation: Republic" >= 0
+				"reputation: Free Worlds" >= 0
+				"reputation: Syndicate" >= 0
+			
+			`	"It doesn't help that you've pissed off the law, either," someone says, not sounding too happy.`
+			label recognition
+			`	More of the patrons of the bar begin to notice that you've entered and begin chattering amongst themselves. "Quite the force to be reckoned with, I've heard," you overhear one of them mumble.`
+			`	"You might want to watch your back," another says to you. "Plenty of pirates looking to make a name for themselves taking out such a famous captain, and plenty of bounty hunters out for a bounty you might have on your head without even knowing about it."`
+			choice
+				`	"I'll be fine. My ship is strong enough to stand against anything."`
+				`	"Maybe I should go into hiding for a while."`
+					goto hide
+			`	"Oh, I'm sure it is. You wouldn't have gotten this notorious if you weren't able to hold your all."`
+				goto end
+			label hide
+			`	"You'd need to hide for more than just a while, but laying low in a small ship might help to stay off the radar of any bounty hunters."`
+			label end
+			`	After a few more minutes of mumbling, the bar returns to like it was before you entered. The food you ordered arrives and you dig in.`
+				decline
+
+mission "Hunted"
+	invisible
+	landing
+	deadline 30
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		attributes "rim" "south" "north" "dirt belt" "core" "frontier"
+	to offer
+		"combat rating" > 8103
+		"armament detterence" >= 2
+		random < 3
+		not "Hunted: active"
```

Comment:
I was expecting to see "Hunted Warning: offered" as condition here.

Reply:
Given that "Hunted" and "Hunted Warning" offer on the same planets, odds are you'll be seeing the warning before seeing any hunted fleet. If a player never goes to the spaceport though, then they'd never get warned about being hunted for being notorious, and therefore actually get any bounty hunters, despite still being notorious.
This is here though so that some unlucky soul doesn't end up stacking multiple Hunted missions at once.

Reply:
If you feel that the warning upfront is desired, then let's make sure the warning also shows for players that never visit the spaceport (and pops-up at landing, not at a spaceport visit to ensure that the hunters cannot be avoided by never visiting the spaceport).

Reply:
I don't see the warning as necessary to be given before hunted fleets show up, only that the warning exists and is readily available. The warning about raid fleets for example is only given if you accept James' help at the beginning, and then only if you bought the Star Barge. The warning exists, but it doesn't need to be received in order for raid fleets to show up. If a player never visits the spaceport to explore the world, it's their own loss when they never bump into someone telling them what's happening.